src/test/basics/base_uint_test.cpp

```diff
@@ -54,9 +74,14 @@ struct base_uint_test : beast::unit_test::suite
         {
             BEAST_EXPECT(d == ++t);
         }
-        checkHash(u, "25996DBE31A04FF03E4E63C9C647AC6D459475E2845529238F3D08F4A37EBC54");
 
-        test96 v(~u);
+        nonhash<96> h;
+        hash_append(h, u);
+        test96 w {std::vector<std::uint8_t>(h.data_.begin(), h.data_.end())};
+        BEAST_EXPECT(w == u);
```

Comment:
Would you mind using a more descriptive variable name or leaving a comment about what this test is testing?